hubblestack_nebula/hubblestack_nebula_queries.yaml

```diff
@@ -54,10 +54,15 @@ day:
   - query_name: system_info
-    query: SELECT t.unix_time AS query_time, si.uuid, si.cpu_type, si.cpu_subtype, si.cpu_brand, si.cpu_physical_cores, si.cpu_logical_cores, si.hardware_vendor, mi.memory_total, mi.memory_free, mi.swap_total, mi.swap_free, pi.vendor AS bios_vendor, pi.version AS bios_version, pi.date AS bios_date, pi.revision AS bios_revision, (SELECT average FROM load_average WHERE period = '1m') AS load_1m, (SELECT average FROM load_average WHERE period = '5m') AS load_5m, (SELECT average FROM load_average WHERE period = '15m') AS load_15m FROM system_info AS si, memory_info AS mi, platform_info AS pi, time AS t;
+    query: SELECT t.unix_time AS query_time, si.uuid, si.cpu_type, si.cpu_subtype, si.cpu_brand, si.cpu_physical_cores, si.cpu_logical_cores, si.hardware_vendor, si.hardware_serial, si.hardware_model , mi.memory_total, mi.memory_free, mi.swap_total, mi.swap_free, pi.vendor AS bios_vendor, pi.version AS bios_version, pi.date AS bios_date, pi.revision AS bios_revision, (SELECT average FROM load_average WHERE period = '1m') AS load_1m, (SELECT average FROM load_average WHERE period = '5m') AS load_5m, (SELECT average FROM load_average WHERE period = '15m') AS load_15m FROM system_info AS si, memory_info AS mi, platform_info AS pi, time AS t;
   - query_name: hubble_log_file
     query: SELECT t.unix_time AS query_time, f.path AS object_path, f.uid AS user_id, f.gid AS group_id, u.username AS user, g.groupname AS 'group', f.mode AS file_acl, f.size AS file_size, f.block_size AS blocks_size, m.path AS mount_path, max(length(m.path)) AS mount_path_length, m.blocks_free, m.inodes_free, m.type AS type, (m.blocks_size*m.blocks_free) AS bytes_free, (m.blocks_size*m.blocks) AS total_bytes, round((1-(m.blocks_free*1.0/m.blocks))*100,2) AS percentage_utilized FROM file AS f JOIN mounts AS m ON (m.type!='rootfs' AND instr(f.path, m.path)=1) LEFT JOIN users AS u ON u.uid=f.uid LEFT JOIN groups AS g ON g.gid=f.gid, time AS t WHERE f.path='/var/log/hubble' GROUP BY f.path;
   - query_name: ami_info
     query: SELECT t.unix_time AS query_time, filename AS image_info FROM file LEFT JOIN time AS t WHERE directory='/etc/image_info';
   - query_name: profile_version
     query: SELECT t.unix_time AS query_time, '2017-11-1' AS version FROM time AS t;
+  - query_name: pci_devices
+    query: SELECT t.unix_time as query_time , pi.pci_slot as pci_slot , pi.pci_class as pci_class , pi.driver as driver , pi.vendor as vendor , pi.vendor_id as vendorid , pi.model as model , pi.model_id as model_id FROM pci_devices as pi LEFT JOIN time AS t;
+  - query_name: hardware_events
+    query: SELECT t.unix_time as query_time , he.action as action , he.type as type , he.time as time , he.model as model  FROM hardware_events as he LEFT JOIN time AS t;
```

Comment:
Unfortunately we can't use _hardware_events_ table from osquery due to the way Hubble works today. So this query has to be removed.